Update changelog for 5.13 (#21616)
Ultimaker/Cura: resources/texts/change_log.txt

[5.13]

* New features and improvements:
- Added the new UltiMaker Factor 4+ printer, including profiles

* New Settings:
- Introduced Infill Start/End Preference settings that give control over the starting location of infill

> - Added Support Infill Line Multiplier, Support Lightning Angle, Support Lightning Overhang Angle, Support Lightning Prune Angle, and Support Lightning Straightening Angle, which together allow you to add and control lightning infill generated for support structures

Reviewer: I would mention the `Infill Line Multiplier` after the Lightling infill, with maybe a comment that it performs very well in combination with the Lightling. But it could also be used for other infills.

Author: What you think about
```suggestion
- Added Support Infill option "Lightning", and in addition the following supporting settings: Support Lightning Angle, Support Lightning Overhang Angle, Support Lightning Prune Angle, and Support Lightning Straightening Angle, which together allow you to add and control lightning infill generated for support structures
- Added Support Infill Line Multiplier which multiplies the number if infill lines, using this setting equal to 2 in combination with lightning support results in a single extrusion line per island
```
?

Reviewer: Yep, love it !

- Introduced Minimum Hole Area in Brim Support, which filters out small holes in the support brim, resulting in a more consistent brim
- Added Temperature Delta Top Surface, which allows you to print top surfaces at a different temperature
- Added more options to Retract Before Outer Wall to give more control over retraction behavior when traveling to the outer wall

* Bug fixes:
- Adding support blockers is no longer assigned to the first extruder if the first extruder is disabled
- Fixed an issue where thin support structures would be generated within tree support
- Fixed an issue where the tree support interface layer would be missing
- Randomized infill now also works for infill skin support
- Fixed an issue where Keep or Discard profile user preference would be ignored

> - Overhanging wall speed was previously applied too aggressively, resulting in extremely slow printing in overhanging areas

Reviewer: ```suggestion
- Fixed an issue where overhanging wall speed was previously applied too aggressively, resulting in extremely slow printing in overhanging areas
```

Author: I dislike all entries starting with "Fixed an issue where..." so I try to mix it up where I can

> - Travel speed was applied to layer switches, resulting in high-speed Z-travels

Reviewer: ```suggestion
- Fixed an issue where travel speed was applied to layer switches, resulting in high-speed Z-travels
```

Author: I dislike all entries starting with "Fixed an issue where..." so I try to mix it up where I can

Reviewer: I actually agree, that was just for consistency...

- Fixed an issue where certain 3MF files would incorrectly load with an invalid setting configuration
- Fixed an issue where certain setting combinations would skip parts of the skin and walls sections
- Resolved an issue where slicing would fail when both Raft build plate adhesion and Support Lightning Infill were enabled
- Fixed an issue where overhang speed would be applied inconsistently when both Split Infill and Overhang Speed settings are enabled
- Fixed an issue where the profile has changes notification would overlap with UI elements

> - Fixed an issue where certain setting configurations would cause small line segments to be generated for the prime tower

Reviewer: "small segments" doesn't sound bah enough to me 😄 
```suggestion
- Fixed an issue where certain setting configurations would cause a lot of tiny line segments to be generated for the prime tower
```

Author: I find "a lot of tiny" rather colloquial language

Reviewer: Yes maybe :smiley:
What about this ?
```suggestion
- Fixed an issue where certain setting configurations would cause a large amount of small line segments to be generated for the prime tower
```
